test/dumb-jump-test.el

```diff
@@ -424,14 +424,14 @@
 (ert-deftest dumb-jump-context-point-test ()
   (let* ((sentence "mainWindow.loadUrl('file://')")
          (func "loadUrl")
-         (ctx (dumb-jump-get-point-context sentence func 15)))
+         (ctx (dumb-jump-get-point-context sentence func 11)))
```

Comment:
I am not sure I understand why it's necessary to change `cur-pos`. It seems to imply that it won't now work _anywhere_ on the current symbol, or am I missing something here?

Reply:
You are correct that it won't work anywhere in the current symbol. But the code will never place it inside the symbol. The `look-for` symbol comes from the code below

https://github.com/jacktasia/dumb-jump/blob/b47774bc167fbd5d336700a17a66f8bc2b7cec46/dumb-jump.el#L2139-L2140

And we know that the `cur-pos` hint will always be the start of the `dumb-jump-get-point-symbol` So there is no way for `cur-pos` to be anywhere in the symbol, it will always be the left of it. This was true even with the old code, but the old `dumb-jump-get-point-context` could handle the case where the hint was inside the symbol even though that case couldn't happen.